kcoin/genesis/genesis_generator_test.go

```diff
@@ -0,0 +1,305 @@
+package genesis
+
+import (
+	"encoding/json"
+	"flag"
+	"github.com/kowala-tech/kcoin/common"
+	"github.com/kowala-tech/kcoin/core"
+	"github.com/kowala-tech/kcoin/params"
+	"github.com/stretchr/testify/assert"
+	"io/ioutil"
+	"math/big"
+	"path/filepath"
+	"testing"
+)
+
+var update = flag.Bool("update", false, "update .golden files")
+
+func TestItFailsWhenRunningHandlerWithInvalidCommandValues(t *testing.T) {
```

Comment:
Just a personal preference, I like to see positive tests first, errors/edge cases later. Main reason is to open a test file and the first thing to see is a kind of example of how this works, if you need more detail you can read more.

Reply:
Interesting point of view, never thought about it, has sense. I normally start with the edge cases than the happy path, that is why it appears in this order. I will change it.